x-pack/plugins/siem/public/components/localized_date_tooltip/index.tsx

```diff
@@ -25,10 +25,15 @@ export const LocalizedDateTooltip = pure<{
           </EuiFlexItem>
         ) : null}
         <EuiFlexItem grow={false}>
-          <FormattedRelative data-test-subj="humanized-relative-date" value={date} />
+          <FormattedRelative
+            data-test-subj="humanized-relative-date"
+            value={moment.utc(date).toDate()}
+          />
         </EuiFlexItem>
         <EuiFlexItem data-test-subj="with-day-of-week" grow={false}>
-          {moment(date)
+          xavier
```

Comment:
Nice try ..... I see you trying sneak your name into the UI. I would recommend removing your name. :-)